src/api/index.js

```diff
@@ -113,10 +113,17 @@ export function getPipeline({ name, namespace }) {
   return get(uri);
 }
 
-export function getPipelineRuns({ namespace, pipelineName } = {}) {
+function getFilterValues(value) {
+  return value.tag;
+}
+
+export function getPipelineRuns({ namespace, pipelineName, filters } = {}) {
   let queryParams;
+  const pipelineNameArray = [`tekton.dev/pipeline=${pipelineName}`];
+  const filterValues = filters.map(getFilterValues);
```

Comment:
the filters should be passed to the API function already formatted correctly

Reply:
Turns out this formatting wasn't needed at all 😄